edb/server/compiler/dbstate.py

```diff
@@ -748,17 +737,16 @@ class ProposedMigrationStep(NamedTuple):
 
     def to_json(self) -> Dict[str, Any]:
         return {
-            'statements': [{'text': stmt} for stmt in self.statements],
-            'confidence': self.confidence,
-            'prompt': self.prompt,
-            'prompt_id': self.prompt_id,
-            'data_safe': self.data_safe,
-            'required_user_input': list(self.required_user_input),
+            "statements": [{"text": stmt} for stmt in self.statements],
+            "confidence": self.confidence,
+            "prompt": self.prompt,
+            "prompt_id": self.prompt_id,
+            "data_safe": self.data_safe,
+            "required_user_input": list(self.required_user_input),
```

Comment:
It's probably not worth undoing but all these automated changes make this diff pointlessly noisy

Reply:
You clearly have something enabled, plenty of changes here just change `'` to `"`

Reply:
Must have pressed a "format file" shortcut or something. It's just this file.